person.rb

```diff
@@ -0,0 +1,24 @@
+class Person
+  def initialize(age, name = 'Unknown', parent_permission: true)
+    @id = Random.rand(1..1000)
+    @name = name
+    @age = age
+    @parent_permission = parent_permission
+  end
+  attr_accessor :name, :age
+  attr_reader :id
+
+  def can_use_services?
+    of_age? || @parent_permission
+  end
+
+  private
+
+  def of_age?
+    @age >= 18
+  end
+end
+
+person = Person.new(18, 'John', parent_permission: true)
+puts person.can_use_services?
+puts person.id
```

Comment:
- [ ] [OPTIONAL] Kindly remove the console logs in order to have a clean code base.